src/sql/command.py

```diff
@@ -120,7 +121,45 @@ def return_result_var(self):
         return self.parsed["return_result_var"]
 
     def _var_expand(self, sql, user_ns):
-        return Template(sql).render(user_ns)
+        template_render = Template(sql).render(user_ns)
+        template_render = self.remove_leading_parentheses(template_render)
+        return template_render
+
+    def remove_leading_parentheses(self, rendered_sql_command):
```

Comment:
I'm confused about this logic (looks very complex).

seems like we only need to strip whitespace and then remove the first and last character if they are parenthesis.

is this function covering other edge cases? if so, we need to document them here so it's clear

Reply:
@edublancas It can create a bad query in case the query is something like `(...) UNION (...)`. I have added notes to the function documentation [here](https://github.com/ploomber/jupysql/pull/934/files#diff-769097f5e36be57ddec10c78df0900db8c1e254f5485bc94367aa7fcb07991c5R140-R150).